src/controllers/user.js

```diff
@@ -115,3 +115,21 @@ export const getStudents = async (req, res) => {
     return sendMessageResponse(res, 500, 'Server Not Found')
   }
 }
+
+export const addUserToCohort = async (req, res) => {
+  const { student_Id: studentId } = JSON.parse(req.body.body)
+  const { id } = req.params
+  if (!id) {
+    return sendMessageResponse(res, 500, 'Cohort Not Found')
+  }
+  try {
+    const addedStudentToCohort = await User.updateStudentByCohort(id, studentId)
+    return sendDataResponse(
+      res,
+      200,
+      `Student ID:${addedStudentToCohort.id} successfully added to Cohort ID:${addedStudentToCohort.cohortId}`
+    )
+  } catch (e) {
+    return sendMessageResponse(res, 500, 'Server Not Found')
```

Comment:
Here I'd suggest two changes. Firstly, it would be a good idea to log whatever error has happened. We can use the `e.message` property to print the error that we caught. 

Secondly, I think rather than returning the message "Server Not Found" We could also return the `e.message`.

Reply:
I've changed 'Server Not Found' to e.message, is this what you meant?